lib/src/util/map_util.dart

```diff
@@ -56,13 +56,13 @@ Map getPropsToForward(Map props, {
   }
 
   if (onlyCopyDomProps) {
-    new List.from(propsToForward.keys).forEach((String key) {
-      if (key.startsWith('aria-')) return;
-      if (key.startsWith('data-')) return;
-      if (_validDomProps.contains(key)) return;
+    for (var key in new List.from(propsToForward.keys)) {
```

Comment:
The last time we changed a `forEach` to a `for` loop... we broke a bunch of stuff... can we verify that we didn't do that again here? _(I don't recall specifically what went wrong last time)_

Reply:
I believe it was this same sort of loop too where you had the list manipulation within the declaration.

@clairesarsam-wf found/fixed it so she might recall the specifics.

Reply:
It was due to `return` not being swapped out with `continue`

Reply:
That's right.